Utilites/Tokenizer/tokenize.ps1

```diff
@@ -10,6 +10,8 @@ param
 )
 
 . $PSScriptRoot\Helpers.ps1
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal" 
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common" 

```

Comment:
Are you sure you need to import the `Microsoft.TeamFoundation.DistributedTask.Task.Common` module? `Get-TaskVariable` is defined in `Microsoft.TeamFoundation.DistributedTask.Task.Internal` and I think it should be enough to just import `Microsoft.TeamFoundation.DistributedTask.Task.Internal` for `Get-TaskVariable` to work.